nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/src/main/java/org/apache/nifi/event/transport/netty/NettyEventServer.java

```diff
@@ -20,23 +20,52 @@
 import io.netty.channel.EventLoopGroup;
 import org.apache.nifi.event.transport.EventServer;
 
+import java.util.concurrent.TimeUnit;
+
 /**
  * Netty Event Server
  */
 class NettyEventServer implements EventServer {
+    static final long SHUTDOWN_QUIET_PERIOD = 2000;  // netty default
+    static final long SHUTDOWN_TIMEOUT = 15000;  // netty default
+    static final TimeUnit SHUTDOWN_TIME_UNIT = TimeUnit.MILLISECONDS;
```

Comment:
In light of the duplication of these properties between `NettyEventSender` and `NettyEventServer`, it seems like they should be refactored to a shared location. Perhaps an `enum` named something like `ShutdownDuration` with `TIMEOUT` and `QUIET_PERIOD` properties?

Reply:
Added enums for quiet period and for timeout.